Content.Shared/Mech/EntitySystems/SharedMechSystem.cs

```diff
@@ -388,6 +390,14 @@ public bool TryEject(EntityUid uid, MechComponent? component = null)
         RemoveUser(uid, pilot);
         _container.RemoveEntity(uid, pilot);
         UpdateAppearance(uid, component);
+
+        // Frontier - Make NPC AI attack Mechs
+        if (TryComp<MobStateComponent>(uid, out var state))
+            RemComp<MobStateComponent>(uid);
+        if (TryComp<NpcFactionMemberComponent>(uid, out var faction))
+            RemComp<NpcFactionMemberComponent>(uid);
```

Comment:
Again, unused variables.
```suggestion
        if (TryComp<MobStateComponent>(uid, out var _))
            RemComp<MobStateComponent>(uid);
        if (TryComp<NpcFactionMemberComponent>(uid, out var _))
            RemComp<NpcFactionMemberComponent>(uid);
```